controllers/lastpass_controller.go

```diff
@@ -205,9 +179,15 @@ func newSecretForCR(cr *edgelevelv1alpha1.LastPass, secret lastpass.LastPassSecr
 		data["NOTE"] = secret.Note
 	}
 
+	var secretName bytes.Buffer
+	r.SecretNameTemplate.Execute(&secretName, struct {
+		LastPass       *edgelevelv1alpha1.LastPass
+		LastPassSecret lastpass.LastPassSecret
+	}{cr, secret})
+
 	return &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:        cr.Name + "-" + secret.ID,
+			Name:        secretName.String(),
```

Comment:
i haven't used it in a while, just wondering about potential regressions, do you have an example of how it was looking before `cr.Name + "-" + secret.ID` compared to now?

Reply:
I think I remembered why I did this, would there be any potential issues with secret conflicts? See example in the readme `example-lastpass-8190226423897406876`

Reply:
The default template used to generate the secret name is the following one `{{.LastPass.ObjectMeta.Name}}-{{.LastPassSecret.ID}}`, so it replicate the old behavior and should not introduce any breaking change. After this change I get by default names like the following `lp-cred1-7726733720672245904`.

Reply:
Thanks for clarifying this!